[2022-06-03 22:07:57,878] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: UK_COVID19_dag.Scale_UK_Data 2022-06-03T21:54:35.086671+00:00 [queued]>
[2022-06-03 22:07:58,445] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: UK_COVID19_dag.Scale_UK_Data 2022-06-03T21:54:35.086671+00:00 [queued]>
[2022-06-03 22:07:58,449] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2022-06-03 22:07:58,451] {taskinstance.py:1043} INFO - Starting attempt 1 of 2
[2022-06-03 22:07:58,452] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2022-06-03 22:07:58,535] {taskinstance.py:1063} INFO - Executing <Task(PythonOperator): Scale_UK_Data> on 2022-06-03T21:54:35.086671+00:00
[2022-06-03 22:07:58,682] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'UK_COVID19_dag', 'Scale_UK_Data', '2022-06-03T21:54:35.086671+00:00', '--job-id', '14', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/covid_dag-checkpoint.py', '--cfg-path', '/tmp/tmp8tlxzhat', '--error-file', '/tmp/tmpqpakkd7c']
[2022-06-03 22:07:58,696] {standard_task_runner.py:77} INFO - Job 14: Subtask Scale_UK_Data
[2022-06-03 22:07:58,657] {standard_task_runner.py:52} INFO - Started process 110 to run task
[2022-06-03 22:08:00,802] {logging_mixin.py:104} INFO - Running <TaskInstance: UK_COVID19_dag.Scale_UK_Data 2022-06-03T21:54:35.086671+00:00 [running]> on host db1a6c3b41a0
[2022-06-03 22:08:01,337] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Huda
AIRFLOW_CTX_DAG_ID=UK_COVID19_dag
AIRFLOW_CTX_TASK_ID=Scale_UK_Data
AIRFLOW_CTX_EXECUTION_DATE=2022-06-03T21:54:35.086671+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-06-03T21:54:35.086671+00:00
[2022-06-03 22:08:19,256] {python.py:118} INFO - Done. Returned value was: None
[2022-06-03 22:08:19,318] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=UK_COVID19_dag, task_id=Scale_UK_Data, execution_date=20220603T215435, start_date=20220603T220757, end_date=20220603T220819
[2022-06-03 22:08:19,457] {taskinstance.py:1220} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-06-03 22:08:20,038] {local_task_job.py:146} INFO - Task exited with return code 0
